src/Sulu/Bundle/WebsiteBundle/Tests/Unit/Sulu/Bundle/WebsiteBundle/EventListener/AppendAnalyticsListenerTest.php

```diff
@@ -405,10 +392,24 @@ public function testAppendPreview()
             true
         );
 
-        $event = $this->prophesize(FilterResponseEvent::class);
-        $event->getRequest()->shouldNotBeCalled();
-        $event->getResponse()->shouldNotBeCalled();
+        $request = $this->prophesize(Request::class);
+        $request->getRequestFormat()->shouldNotBeCalled();
+        $response = $this->prophesize(Response::class);
+        $response->getContent()->shouldNotBeCalled();
```

Comment:
That does not test the same things as on the left, right? Why is that?

Reply:
the `$event` can not longer be mocked so I need to check that the `$request` and `$response` is not called.